R/style_guides.R

```diff
@@ -166,10 +172,35 @@ create_style_guide <- function(initialize = initialize_attributes,
     token,
     indention,
     # transformer options
-    use_raw_indention
+    use_raw_indention,
+    reindention
   )
 }
 
+
+#' Specify what is re-indented how
+#'
+#' This function returns a list that can be used as an input for the argument
+#' `reindention` of the function [tidyverse_style()]. It features sensible
+#' defaults, so the user can specify deviations from them conveniently without
+#' the need of setting all arguments explicitly.
+#' @param regex_pattern Character vector with regular expression patterns that
+#'   are to be re-indented with spaces.
+#' @param indention The indention tokens should have if they match
+#'   `regex_pattern`.
+#' @param comments_only Whether the `regex_reindention_pattern` should only be
+#'   matched against comments or against all tokens. Mainly added for
+#'   performance.
+specify_reindention <- function(regex_pattern = regex_strcode(),
```

Comment:
Should this default to `regex_none()`, at least for the tidyverse style?

Reply:
Ok.